### PR TITLE
Ng2 router

### PR DESCRIPTION
**Pull request by KorinaGraber**
Converted the components bootstrapper to use the new angular component router. This serves as a spike for using the router in RL21, but will also be a nice change so the bootstrapper still works like the main app once we finish the main router migration.

### CODE REVIEW

**Comment by KorinaGraber** [comment]
New commits add a new button route component and convert the MSI to the angular 2 router

File: index.html [hunk 1]
```diff
@@ -6,17 +6,21 @@
...
+		<base href="/" />
+
 		<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
 		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
 		<link rel="stylesheet" href="/dist/components.css">
 
 		<div>
-			<ts-bootstrapper></ts-bootstrapper>
+			<ts-app></ts-app>
+			<!--<ts-bootstrapper></ts-bootstrapper>-->
```
Comment (KorinaGraber) [inline index.html]: This comment should go away

File: source/components/multiStepIndicator/multiStepIndicator.ts [hunk 2]
```diff
@@ -82,35 +80,39 @@ export class MultiStepIndicatorComponent implements OnInit {
 		});
 	}
 
-	private redirectToState(step: IConfiguredStep): Promise<void> {
-		return this._state.go(step.stateName).then((): void => {
-			this.clearCurrentState();
-			step.isCurrent = true;
+	private redirectToState(step: IConfiguredStep): Observable<void> {
+		return Observable.create(observer => {
```
Comment (KorinaGraber) [inline source/components/multiStepIndicator/multiStepIndicator.ts]: Could use Observable.fromPromise instead

**Review by KorinaGraber — COMMENTED** [review]
A few ideas for improvements

**Comment by arknotts** [comment]
:shipit:

**Pull request merged**